frontend/src/bundles/common/components/input/input.tsx

```diff
@@ -26,22 +31,48 @@ const Input = <T extends FieldValues>({
     isDisabled = false,
     placeholder = '',
 }: Properties<T>): JSX.Element => {
+    const [isMasked, setIsMasked] = useState(false);
```

Comment:
It seems a bit counterintuitive that when isMasked is true, the password is visible instead of being masked.
Also currently the eye icon will be displayed when the password is already visible, which is the wrong condition.